.travis.yml

```diff
@@ -31,7 +31,7 @@ install:
 script:
   - export PYTHONPATH=`pwd`:$PYTHONPATH
   - python src/tests/runtests.py
-  # - nunit-console src/embed_tests/bin/x64/ReleaseMono/Python.EmbeddingTest.dll
+  # - nunit-console src/embed_tests/bin/Python.EmbeddingTest.dll
```

Comment:
Okay, fine by me then, but I'll open an issue for this, we should really start running those tests.